www/docs/mp/index.php

```diff
@@ -40,11 +40,16 @@
 include_once '../api/api_getConstituencies.php';
 
 // Ensure that page type is set
+$allowed_page_types = ['divisions', 'votes', 'policy_set_svg', 'policy_set_png', 'recent', 'register'];
+
 if (get_http_var('pagetype')) {
     $pagetype = get_http_var('pagetype');
 } else {
     $pagetype = 'profile';
 }
+if (!in_array($pagetype, $allowed_page_types)) {
```

Comment:
So the issue here is without this check, the full version of the URL with the constituency in it gets the constituency assigned to the $pagetype - which might be an argument just for using the httpd.conf to manage this? Your call.

Reply:
Ah, right - hmm, I guess the get_person_by_id check redirects if it doesn't match a known thing anyway, so actually this works okay (just with a redirect, rather than a 404 - I just didn't want /mp/N/name/const/random to display a page, which it presuambly still doesn't).

Reply:
Yes - that's right, it redirects back to the main profile page if you add things after the constituency not in this list.